docs/app/parent-child.adoc

```diff
@@ -181,3 +181,11 @@ parent:
     - RUNNER_STOPPED
     - RUNNER_REMOVED
 ----
+
+=== Filtering Events Based on Commands
```

Comment:
This should probably sub-section of "Filtering Events" rather than existing at the same level.